FindOpenGLES.cmake

```diff
@@ -0,0 +1,69 @@
+#.rst:
+# FindOpenGLES
+# ------------
+# Finds the OpenGLES2 library
+#
+# This will define the following variables::
+#
+# OPENGLES_FOUND - system has OpenGLES
+# OPENGLES_INCLUDE_DIRS - the OpenGLES include directory
+# OPENGLES_LIBRARIES - the OpenGLES libraries
+# OPENGLES_DEFINITIONS - the OpenGLES definitions
+#
+# Note:
+# On Windows with angle the *_INCLUDE_DIRS and
+# *_DEFINITIONS are undefined, but are set
+# global by the kodi-angle package.
+
+if(WIN32)
```

Comment:
not used on windows, so drop everything related to it

Reply:
Thought to prepare direct for Matrix, becomes removed.